pkg/awsapi/iam.go

```
@@ -104,12 +109,18 @@ func (i *IAM) CreateRole(ctx context.Context, req IAMRoleRequest) (*IAMRoleRespo
 		return &IAMRoleResponse{}, err
 	}
 
+	//Attach inline role policy
 	err = i.AddPermissionBoundary(ctx, req)
 
 	if err != nil {
 		return &IAMRoleResponse{}, err
 	}
 
+	//Attach managed role policy
+	for _, policy := range ManagedPolicies {
+		err = i.AttachManagedRolePolicy(ctx, policy, req.Name)
```

You should return the error and log the error